test/aws-xray-sdk/tc_context.rb

```diff
@@ -36,7 +37,6 @@ def test_runtime_error
   def test_log_error
     context = XRay::DefaultContext.new
     context.clear!
-    context.context_missing = 'LOG_ERROR'
```

Comment:
Since this test is called `test_log_error`, perhaps adding an assertion that context.context_missing is LOG_ERROR by default would also make sense

Reply:
Good point. Originally `test_runtime_error` assumed the default value, so I thought to test `test_log_error` assuming the default value as well with the new change. I think it makes sense to incorporate your suggestion, and then add a `test_default_context_missing` test case.